setup.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-createdb invidious
-createuser kemal
-psql invidious < config/sql/channels.sql
-psql invidious < config/sql/videos.sql
-psql invidious < config/sql/channel_videos.sql
-psql invidious < config/sql/users.sql
+createdb -U kemal invidious
+# createuser -U kemal kemal
+psql -U kemal invidious < config/sql/channels.sql
+psql -U kemal invidious < config/sql/videos.sql
+psql -U kemal invidious < config/sql/channel_videos.sql
+psql -U kemal invidious < config/sql/users.sql
```

Comment:
It might make more sense for these commands to be run similar to how they are in the [Dockerfile](https://github.com/omarroth/invidious/pull/186/files#diff-ebacf6f6ae4ee68078bb16454b23247dR3) if possible, since the changes here break normal setup. If that's not possible, I'd move these changes into a separate file for docker to use.

Reply:
this line in setup.sh: `createuser kemal` - wouldn't it create the user without password? Because then in config.yml we have user kemal with password kemal, so I'm surprised it's even working for you... For the automatic install I think I have to change one or the other...

Reply:
You're right. Looks like [here](https://www.postgresql.org/docs/10/static/app-createuser.html) when creating a user password authentication is disabled by default, so any password in `config.yml` appears to work. That's my mistake.

Your approach works fine for the docker installation, so I think I'll mark this as resolved. For the default installation, I think I'll change how the kemal user is created, so a user can change e.g. default password, and add that info to the README.

Reply:
I actually found another approach, where the only change in setup.sh will be

# createuser kemal
psql -c "CREATE USER kemal WITH PASSWORD 'kemal';"

the rest can stay the same, without "-U kemal". I'll push it to this PR soon